src/core/optimization_container.jl

```diff
@@ -1627,7 +1627,7 @@ function add_to_objective_invariant_expression!(
 ) where {T <: JuMP.AbstractJuMPScalar}
     T_cf = typeof(container.objective_function.invariant_terms)
     if T_cf <: JuMP.GenericAffExpr && T <: JuMP.GenericQuadExpr
-        # why not add_to...???
+        # TODO: why not add_to...???
```

Comment:
The `add_to` here usually fails is not allowed by JuMP

Reply:
This is weird, do you have an example? It does not happen in RTS. Maybe it was an old JuMP bug?

Reply:
This only happens when we have data that mixes PWL and Quadratic cost functions.

Reply:
If there is any case I can test with those features I'd be happy to.